rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -890,6 +896,461 @@ private void testSaveOldFederation(
         );
     }
 
+    @Test
+    void saveFederationElection() {
+        ABICallElection electionMock = mock(ABICallElection.class);
+        List<Integer> storageBytesCalls = new ArrayList<>();
+        List<Integer> serializeCalls = new ArrayList<>();
+        Repository repositoryMock = mock(Repository.class);
+        FederationStorageProvider federationStorageProvider = createFederationStorageProvider(repositoryMock);
+
+        try (MockedStatic<BridgeSerializationUtils> bridgeSerializationUtilsMocked = mockStatic(BridgeSerializationUtils.class)) {
+            bridgeSerializationUtilsMocked.when(() -> BridgeSerializationUtils.serializeElection(any(ABICallElection.class))).then((InvocationOnMock invocation) -> {
+                ABICallElection election = invocation.getArgument(0);
+                Assertions.assertSame(electionMock, election);
+                serializeCalls.add(0);
+                return Hex.decode("aabb");
```

Comment:
Let's move this `aabb` string literally to a variable with a meaningful name of what exactly it represents. Wdyt?